compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -137,22 +156,36 @@ private class SkiaBackedTypeface(
  * @param identity Unique identity for a font. Used internally to distinguish fonts.
  * @param data Byte array with loaded font data.
  * @param weight The weight of the font. The system uses this to match a font to a font request
- * that is given in a [androidx.compose.ui.text.SpanStyle].
+ * that is given in a [androidx.compose.ui.text.SpanStyle]. For non-variable fonts, this directly
+ * affects the font style.
  * @param style The style of the font, normal or italic. The system uses this to match a font to a
- * font request that is given in a [androidx.compose.ui.text.SpanStyle].
+ * font request that is given in a [androidx.compose.ui.text.SpanStyle]. For non-variable fonts,
+ * this directly affects the font style.
+ * @param variationSettings Specifies the variation settings for the font. This allows for fine-tuned
+ * control over font characteristics such as weight and style. It can be used to create custom
+ * font variations within a single font file, if the font supports variable axes. By default,
+ * it uses the provided weight and style parameters.
+ *
+ * Note on font style determination:
+ * 1. For non-variable fonts, the font style is defined only by the weight and style parameters.
+ * 2. For variable fonts, the font style is determined solely by the variationSettings.
+ * 3. When weight, style, and variationSettings are all set for variable fonts, the font style
+ *    will be determined by variationSettings, as its default values are derived from weight and style.
  *
  * @see FontFamily
  */
 fun Font(
     identity: String,
     data: ByteArray,
     weight: FontWeight = FontWeight.Normal,
-    style: FontStyle = FontStyle.Normal
+    style: FontStyle = FontStyle.Normal,
+    variationSettings: FontVariation.Settings = FontVariation.Settings(weight, style),
```

Comment:
It breaks binary compatibility for public/stable method. You need to keep the previous method as-is or under `@Deprecated(HIDDEN)`